Procfile

```diff
@@ -1,3 +1,4 @@
 release: STATEMENT_TIMEOUT=180000 bundle exec rails db:migrate
 web: bin/start-pgbouncer bundle exec puma -C config/puma.rb
 worker: bundle exec rails jobs:work
+sidekiq_worker: bundle exec sidekiq -t 25
```

Comment:
Heroku gives process 30 seconds before forcefully killing them. This tells Sidekiq to give jobs 25 seconds to finish after which it will push them back to Redis to be retried after the restart. [More info on Heroku and Sidekiq](https://github.com/mperham/sidekiq/wiki/Deployment#heroku)

Reply:
`-t 25` is actually the default now but I like having it here as a reminder.